.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+  - 3.6
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
+# command to run tests
+script:
+  - flake8 app --statistics --count
+# TODO: implement pytests then delete this line and uncomment the next
+#  - pytest
```

Comment:
Nit: When I initially committed this, I was missing the newline from the end of the file. I'd appreciate it if you'd please pick up my slack and stick that in 😉

Reply:
Added :)